### ISSUE TITLE
HypothesisTests throws overwrite warnings on v0.5 when imported

### ISSUE THREAD

**Issue by tlnagy** [JuliaStats/HypothesisTests.jl#75]
I get errors like these on v0.5 when `using HypothesisTests`:
```
WARNING: Method definition pvalue(HypothesisTests.FisherTLinearAssociation) in module HypothesisTests at /Users/tamasnagy/.julia/v0.5/HypothesisTests/src/circular.jl:139 overwritten at /Users/tamasnagy/.julia/v0.5/HypothesisTests/src/circular.jl:139.
WARNING: Method definition #pvalue(Array{Any, 1}, HypothesisTests.#pvalue, HypothesisTests.FisherTLinearAssociation) in module HypothesisTests overwritten.
WARNING: Method definition pvalue(HypothesisTests.FisherTLinearAssociation) in module HypothesisTests at /Users/tamasnagy/.julia/v0.5/HypothesisTests/src/circular.jl:139 overwritten at /Users/tamasnagy/.julia/v0.5/HypothesisTests/src/circular.jl:139.
WARNING: Method definition #pvalue(Array{Any, 1}, HypothesisTests.#pvalue, HypothesisTests.FisherTLinearAssociation) in module HypothesisTests overwritten.
```

**Comment by simonster** [comment]
Fixed by a964ed55157664c8c8bc3b01cc48f71fef1c90ce

**Comment by tlnagy** [comment]
Thanks!
